google-cloud-pubsub/lib/google/cloud/pubsub/topic.rb

```diff
@@ -132,6 +132,56 @@ def labels= new_labels
           @resource_name = nil
         end
 
+        ##
+        # The Cloud KMS encryption key that will be used to protect access
+        # to messages published on this topic.
+        # For example: `projects/a/locations/b/keyRings/c/cryptoKeys/d`
+        # The default value is `nil`, which means default encryption is used.
+        #
+        # Makes an API call to retrieve the labels values when called on a
+        # reference object. See {#reference?}.
```

Comment:
It looks like this sentence was copied from `labels`.

Reply:
Ugh...

Reply:
Fixed.